etc/adminhtml/system/adyen_settings_ecommerce_payments.xml

```diff
@@ -0,0 +1,123 @@
+            <comment model="Adyen\Payment\Model\Comment\ApiKeyEnding"/>
+        </field>
+        <field id="notification_username" translate="label" type="text" sortOrder="40" showInDefault="1"
+               showInWebsite="1" showInStore="0">
+            <label>Standard notification username</label>
+            <config_path>payment/adyen_abstract/notification_username</config_path>
+        </field>
+        <field id="notification_password" translate="label" type="obscure" sortOrder="50" showInDefault="1"
+               showInWebsite="1" showInStore="0">
+            <label>Standard notification password</label>
+            <backend_model>Magento\Config\Model\Config\Backend\Encrypted</backend_model>
+            <config_path>payment/adyen_abstract/notification_password</config_path>
+        </field>
+        <field id="live_endpoint_url_prefix" translate="label" type="text" sortOrder="60" showInDefault="1"
+               showInWebsite="1" showInStore="0">
+            <label>Live endpoint prefix</label>
+            <config_path>payment/adyen_abstract/live_endpoint_url_prefix</config_path>
+        </field>
+        <field id="advanced_settings" translate="label" sortOrder="70" showInDefault="1" showInWebsite="1">
+            <label>Advanced settings</label>
+            <frontend_model>Magento\Config\Block\System\Config\Form\Field\Heading</frontend_model>
+            <attribute type="shared">1</attribute>
+        </field>
+        <field id="support_link" translate="label" sortOrder="80" showInDefault="1" showInWebsite="1">
+            <label><![CDATA[Before changing these setting, make sure to contact our <a target="_blank" href="https://support.adyen.com/hc/en-us/requests/new">support team</a>]]></label>
+            <frontend_model>Magento\Config\Block\System\Config\Form\Field\Heading</frontend_model>
+            <attribute type="shared">0</attribute>
+        </field>
+        <field id="capture_mode" translate="label" type="select" sortOrder="90" showInDefault="1" showInWebsite="1" showInStore="1">
+            <label>Capture delay</label>
+            <tooltip>Immediate is the default. Set to manual if you want to perform the capture of funds manually later (only affects credit cards and a few alternative payment methods). You need to change this setting as well in Adyen Customer Area => Settings => Merchant Settings => Capture Delay. If you have selected a capture delay of a couple of days in Adyen keep it here on immediate</tooltip>
+            <source_model>Adyen\Payment\Model\Config\Source\CaptureMode</source_model>
+            <config_path>payment/adyen_abstract/capture_mode</config_path>
+        </field>
+        <field id="auto_capture_openinvoice" translate="label" type="select" sortOrder="100" showInDefault="1" showInWebsite="1" showInStore="1">
+            <label>Open invoice capture delay</label>
+            <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
```

Comment:
Let's add this to the functionality ticket: Capture on shipment / Capture immediately / Capture manually?

Reply:
Mention it PW-1714